exalearn-ui/src/app/components/new-grammar/new-grammar.component.ts

```diff
@@ -50,24 +56,27 @@ export class NewGrammarComponent implements OnInit {
 		} else {
 			this.rightAnswer = true;
 			this.load = false;
-		}
+		} */
 
 		//TODO: delete code above when server is ready
 		// and uncomment code lower
-		/* this.ncService.setIsCorrectProperty(this.form);
+		this.ncService.setIsCorrectProperty(this.form);
 		this.load = true;
 		if (this.ncService.rightAnswerSelected()) {
 			this.rightAnswer = false;
 			void this.apiServise
 				.postRequest(API_PATH.NEW_GRAMMAR, this.form.value)
 				.then(() => {
+					this.notificationService.successPopUp();
 					this.grammarForm.resetForm();
-					this.load = false;
 				})
-				.catch(() => console.log('Sorry something wrong((('));
+				.catch(() => this.notificationService.errorPopUp('Sorry. Something went wrong'))
+				.finally(() => {
+					this.load = false;
```

Comment:
So, do we want to make `this.load = false` in every case? Or do we want to do this only on error falls?